families/config/sawtooth_config/processor/handler.py

```diff
@@ -318,6 +318,41 @@ def _append_setting_entry(state, key, value):
                     key, old_value, value)
 
 
+def _remove_setting_entry(state, key, value):
```

Comment:
This function removes a _value_ from a setting.  It doesn't actually remove the setting.  Should it remove the setting if the only or last value is removed or should this function continue doing what it is doing (i.e., when removing the last value the setting is an empty string) and then a new remove setting is created?